Docker-files/requirements.txt

```diff
@@ -0,0 +1,17 @@
+numpy==1.25.1
```

Comment:
Why is there a different `requirements.txt` and not the same one that kaldo uses?

Reply:
Good point. I will update the requirments.txt to be kaldo's. I will go resolve any dependency issues and tell you if that is the case.